.github/workflows/release.yml

```diff
@@ -22,17 +35,59 @@ jobs:
           go-version: '>=1.20'
           check-latest: true
           cache: true
-      - name: Download syft binary
-        run: curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin
-      - name: Run syft
-        run: syft version
+      
+      - name: Get Tag
+        id: get_tag
+        run: echo "LATEST_TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo 'v0.0.1')" >> $GITHUB_ENV
+
+      - name: Set up Python
```

Comment:
Why do we need this, we are not pushing this SBOM to interlynk. Thats already done in other actions.

Reply:
Okay.